.travis.yml

```diff
@@ -0,0 +1,19 @@
+os:
+  - linux
+  - osx
+
+# Ubuntu 14.04 Trusty support, to get newer cmake and compilers.
+sudo: required
+dist: trusty
+
+language: cpp
+
+script:
+  - mkdir out && cd out
```

Comment:
Please use `build` for the build directory name. That's a more commonly used name and you find it in the CMake examples/usage tutorial.